lib/easypost/util.rb

```diff
@@ -7,9 +7,11 @@ module EasyPost::Util
   BY_PREFIX = {
     'ak' => EasyPost::ApiKey,
     'adr' => EasyPost::Address,
+    'bank' => EasyPost::PaymentMethod,
```

Comment:
There is no such thing as a `PaymentMethod` object. We should not include these here

Reply:
There is an `EasyPost::PaymentMethod` class. The class also exists in other client libs:
https://github.com/EasyPost/easypost-java/blob/master/src/main/java/com/easypost/model/PaymentMethod.java

Reply:
It exists in other type-strict languages because it must. In dynamic languages it's not needed, nor does it actually line up with our API spec. We should have 1:1 mappings without extra models where possible.

Reply:
I would consider it an anti-pattern that some JSON bodies get deserialized into an explicit class instance, while others simply get returned as a generic object.

Reply:
My only concern is that we give people the wrong idea (hey this object is in your libs but I can't find it in your docs).